src/api/files/service.js

```diff
@@ -1,28 +1,35 @@
+import { S3Client, GetObjectCommand } from '@aws-sdk/client-s3'
+import { getSignedUrl } from '@aws-sdk/s3-request-presigner'
 import Boom from '@hapi/boom'
+import argon2 from 'argon2'
 import { MongoServerError } from 'mongodb'
 
 import * as repository from './repository.js'
 
+import { config } from '~/src/config/index.js'
 import { createLogger } from '~/src/helpers/logging/logger.js'
 
 const logger = createLogger()
+const s3Region = config.get('s3Region')
 
 /**
  * Accepts file status into the forms-submission-api
  * @param {UploadPayload} uploadPayload
  */
 export async function ingestFile(uploadPayload) {
-  const { formId } = uploadPayload.metadata
+  const { retrievalKey } = uploadPayload.metadata
   const { file: fileContainer } = uploadPayload.form
 
+  const hashed = await argon2.hash(retrievalKey)
+
   try {
     await repository.create({
-      formId,
-      ...fileContainer
+      ...fileContainer,
+      retrievalKey: hashed
     })
   } catch (err) {
     if (err instanceof MongoServerError && err.errorResponse.code === 11000) {
-      const error = `File ID '${fileContainer.fileId}' for form ID '${formId}' has already been ingested`
+      const error = `File ID '${fileContainer.fileId}' has has already been ingested`
```

Comment:
2 * has :)

Reply:
Fixed in [9f4f60e](https://github.com/DEFRA/forms-submission-api/pull/8/commits/9f4f60e31dce89fac9821bddf5d1f17d07a07fb2), thanks.